src/lib/fauna.mjs

```diff
@@ -87,9 +87,24 @@ export const runQuery = async ({
       url: /** @type {string} */ (url), // We know this is a string because we check for !url above.
       secret: /** @type {string} */ (secret), // We know this is a string because we check for !secret above.
     });
+  const logger = container.resolve("logger");
   // Run the query.
   return _client
     .query(query, { ...defaultV10QueryOptions, ...options })
+    .then((result) => {
+      logger.debug(
+        `Fauna v10 query result: ${JSON.stringify(result, null, 2)}`,
+        "client",
+      );
+      return result;
+    })
+    .catch((err) => {
+      logger.debug(
+        `Fauna v10 query error: ${JSON.stringify(err, null, 2)}`,
```

Comment:
Is it possible that err would not be valid JSON?

Reply:
Suspect so - fetch timeouts are often circular. We could try several strategies in a fallback w/try catch? Maybe:
- JSON.stringify
- util.inspect
- toString

Reply:
the caught error should always be an object of some sort which stringify can handle. If we wanted JSON.parse here that would be more problematic